pandas/_libs/parsers.pyx

```diff
@@ -1439,8 +1439,8 @@ cdef _categorical_convert(parser_t *parser, int64_t col,
             COLITER_NEXT(it, word)
 
             if na_filter:
-                if kh_get_str_starts_item(na_hashset, word):
                 # is in NA values
+                if kh_get_str_starts_item(na_hashset, word):
```

Comment:
the linter wants comments before the `if`?

Reply:
or just an indented comment - I've gone with that, thanks